src/vue/forms/SubmitTradeOfferForm.vue

```diff
@@ -131,14 +212,24 @@ export default {
       return this.accountBalances
         .find(item => item.asset === this.offer.quoteAssetCode)
     },
+    formQuoteAmount () {
+      return MathUtil.multiply(this.form.price, this.form.baseAmount)
+    },
+    totalValue () {
+      return +this.formQuoteAmount ? this.formQuoteAmount : ''
```

Comment:
what is the point of this?

Reply:
it's for displaying a sum in filed it kindda instead  of value for field and + readonly status => Just filed whom we can't to edit or add something other